src/test/run-pass/const-int-saturating-arith.rs

```diff
@@ -0,0 +1,33 @@
+#![feature(const_saturating_int_methods)]
```

Comment:
To fix the build failure, please add the following line
```
// ignore-emscripten no i128 support
```
to the very beginning of this file.